routers/api/v1/misc/markdown.go

```diff
@@ -26,9 +26,9 @@ func Markdown(ctx *context.APIContext, form api.MarkdownOption) {
 
 	switch form.Mode {
 	case "gfm":
-		ctx.Write(markdown.Render([]byte(form.Text), form.Context, nil))
+		ctx.Write(markdown.Render([]byte(form.Text), ctx.Repo.RepoLink, nil))
```

Comment:
I'm not quite sure about this one. I can't recall doing that. Thoughts? Good or bad?